plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -2057,3 +2097,466 @@ describe('PersonState.update()', () => {
         })
     })
 })
+
+describe('person id overrides', () => {
+    // For some reason these tests failed if I ran them with a hub shared
+    // with other tests, so I'm creating a new hub for each test.
+    let hub: Hub
+    let closeHub: () => Promise<void>
+    let teamId: number
+    const timestamp = DateTime.fromISO('2020-01-01T12:00:05.200Z').toUTC()
+
+    beforeAll(async () => {
+        ;[hub, closeHub] = await createHub({})
+    })
+
+    beforeEach(async () => {
+        const organizationId = await createOrganization(hub.db.postgres)
+        teamId = await createTeam(hub.db.postgres, organizationId)
+        // jest.useFakeTimers({ advanceTimers: 50 })
+    })
+
+    afterEach(() => {
+        jest.clearAllTimers()
+    })
+
+    afterAll(async () => {
+        await closeHub()
+    })
+
+    async function updatePersonStateFromEvent(event: Partial<PluginEvent>, ts = '', mergeAttempts = 3) {
```

Comment:
just make it 0 by default - makes it easier to debug tests

Reply:
That would cause all other tests that expects this to retry to fail. I rather not keep the default.

Reply:
which other tests expect that? I don't think there are any

Reply:
I don't know, but have you checked all of them? Because I have not.